### ISSUE TITLE
Combine define generate Help file with Docfx 

### ISSUE THREAD

**Issue by chuongmep** [DynamoDS/DynamoWishlist#236]
If this is an issue for Dynamo Core, please submit your issue on the [Dynamo Core issues page](https://github.com/DynamoDS/Dynamo/issues) or if this is an issue for Dynamo for Revit, please post your issue on the [Dynamo for Revit Issues page](https://github.com/DynamoDS/DynamoRevit/issues).

If this issue is **not** a Wishlist request, please check the [Dynamo forum](https://forum.dynamobim.com/), and start a thread there to discuss your issue.

## What improvement/feature would you like to see in Dynamo?

Example I have this document online : https://chuongmep.github.io/OpenMEP/api/OpenMEP.ConnectorManager.ConnectorManager.html
With Method defind for pictute example : https://github.com/chuongmep/OpenMEP/blob/ec4dee35ac75a515e158f65842ef9f72a2c28df8/OpenMEP/ConnectorManager/ConnectorManager.cs#LL19C5-L19C5

I don't want defind two time one for help file and one for document online, view this isssue : https://github.com/DynamoDS/Dynamo/issues/13648#issuecomment-1465170738

## How would you see this improvement/feature working? 

- User just need defind same format with [docfx](https://github.com/dotnet/docfx) and then can resolve problem for both

## Please provide any sketch/mock-up or video of your requested improvement/feature

Example: 

```cs
    /// <summary>
    /// return connector manager of element
    /// </summary>
    /// <param name="element">element</param>
    /// <returns name="ConnectorManager">Autodesk.Revit.DB.ConnectorManager</returns>
    /// <example>
    /// ![](../OpenMEPPage/connectormanager/dyn/pic/ConnectorManager.GetConnectorManager.png)
    /// </example>
```
/// ![](../OpenMEPPage/connectormanager/dyn/pic/ConnectorManager.GetConnectorManager.png)

This will be generate same with docfx include code, picture, markdown,...

**Comment by Amoursol** [comment]
@chuongmep just to confirm, you are looking for Dynamo to **also** support the .docfx file format as well as the current Markdown approach? 😊

**Comment by chuongmep** [comment]
@Amoursol yes, at the moment my process break with Markdown define in dynamo, it is not same and I don't have any way to connect document guideline.

**Comment by Amoursol** [comment]
@mjkkirschner any reason why we couldn't also support the docfx format? Assuming we'll also need to add it to the HTML sanitizer workflow? So just a time-cost?

**Comment by mjkkirschner** [comment]
markdown is simpler and wider adopted than docfx, docfx requires use of a library and it generates a static site on the other end, unclear if it's appropriate for our use case without investigation.

It may perform some kind of sanitization as well but I have no idea.

**Comment by chuongmep** [comment]
Docfx also support markdown when I insert a path markdown to example, why we can see this is a not wider adopted ? I can write a acticle to explain for one node and link with online documentation.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by johnpierson** [comment]
Closing as part of a wishlist triage (stale / no longer applicable).

Node help uses sanitized Markdown for security reasons, and the team noted that DocFX support is not a good fit. Package authors can keep publishing DocFX sites independently and link to them from their Markdown help.

If this is still relevant to you in current Dynamo, please comment and we'll reopen it.
